packages/bot-web-ui/src/components/dashboard/dashboard-component/dashboard-component.tsx

```diff
@@ -68,6 +65,16 @@ const DashboardComponent = observer(({ handleTabChange }: TMobileIconGuide) => {
     const has_dashboard_strategies = !!dashboard_strategies?.length;
     const is_mobile = isMobile();
 
+    React.useEffect(() => {
+        //on dashbord umount fire close event for rudderstack
+        return () => {
+            RudderStack.track('ce_bot_builder_form', {
```

Comment:
looks like this for dashboard, please fill it proper for `event: 'ce_bot_builder_form'`